### PR TITLE
add slides

### PR DESCRIPTION
**Pull request by Bfinity**
Something's wrong with the slides... not sure what. Battery dead. I'll try to fix it in the morning.

### CODE REVIEW

**Comment by Bfinity** [comment]
I fixed this!

**Comment by kyleofori** [comment]
This is looking good so far. Interested to see what will go in the "Making It Work" section. 

If it's cool, I'd like to pull your "Common Conditionals" section into the part of the presentation I made, UNLESS you were thinking the students only need a little bit of conditionals knowledge for the time being. (If so, let's discuss.)

File: Class Presentations/reveal-presentations/MachineOperations.html [hunk 1]
```diff
@@ -0,0 +1,520 @@
...
+			```javascript
+			function logicalOperator(x, y){
+				if(x == y){
+					console.log(“They are equal”);
+				}
+				else if(x > y){
+					console.log(“x is greater than y”);
+				}
+				else if(x < 2 && y > x){
+				console.log(“x is greater than 2 AND y is greater than x);}
```
Comment (kyleofori) [inline Class Presentations/reveal-presentations/MachineOperations.html]: This should read "x is less than two"

File: Class Presentations/reveal-presentations/MachineOperations.html [hunk 2]
```diff
@@ -0,0 +1,520 @@
...
+				<input type="checkbox" id="goal2" checked/>
+				<label for="goal2">Development Planning</label>
+			</li>
+			<li>
+				<input type="checkbox" id="goal3" checked/>
+				<label for="goal3">Common Conditionals</label>
+			</li>
+			<li>
+				<input type="checkbox" id="goal4"/>
+				<label for="goal4">Making It Work</label>
```
Comment (kyleofori) [inline Class Presentations/reveal-presentations/MachineOperations.html]: Is the `Making It Work` section where they build out the calculator?

**Comment by kyleofori** [comment]
There was a code review using if, else if, and else that came before the section on conditionals. Would it be useful to move the code review until after the conditionals, or did you want them to struggle a bit?

**Comment by kyleofori** [comment]
Last thing: can we toss this link to the student survey in there? We can e-mail it out, too, and give them time to complete it during the last few minutes of class. http://goo.gl/forms/2YG2Bqs6fz

File: Class Presentations/reveal-presentations/MachineOperations.html (hunk 1 above)
Comment (Bfinity) [inline Class Presentations/reveal-presentations/MachineOperations.html]: Fixed.

File: Class Presentations/reveal-presentations/MachineOperations.html (hunk 2 above)
Comment (Bfinity) [inline Class Presentations/reveal-presentations/MachineOperations.html]: Yes and I created another exercise called MachineOps

**Comment by Bfinity** [comment]
I fixed the date, updated the calc exercise and added another. I also added slides for them and another slide for the survey.

**Pull request merged**